src/lib/eina/eina_debug_cpu.c

```diff
@@ -80,11 +80,8 @@ _sysmon(void *data EINA_UNUSED, Eina_Thread thr EINA_UNUSED)
              for (i = 0; i < _eina_debug_thread_active_num; i++)
                {
                   // get the correct clock id to use for the target thread
-                  #ifndef _WIN32
-                     pthread_getcpuclockid(self, &cid);
-                  #endif
-               //   pthread_getcpuclockid
-                     (_eina_debug_thread_active[i].thread, &cid);
+                  pthread_getcpuclockid(self, &cid);
+                  (_eina_debug_thread_active[i].thread, &cid);
```

Comment:
What is this line doing?

Reply:
oh wait

Reply:
This line is doing nothing because of my attetion, sorry